cmd/fluent-watcher/fluentd/main.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	defaultBinPath      = "/usr/bin/fluentd"
+	defaultBinPath      = "/usr/local/bundle/bin/fluentd"
```

Comment:
The default path for the `fluentd` binary (both amd64 and arm64) is `/usr/local/bundle/bin` in the upstream images (which we are now using).